docs/document/docs/guides/tabs-customzation.md

```diff
@@ -0,0 +1,200 @@
+Finally, we call the `createTabs()` function to create the tabs and add them to the specified space.
+
+```javascript
+function initCustomComponentTabs() {
+  const salesRecordTable = new Kuc.ReadOnlyTable({
+    label: 'Sales Record',
+    columns: [
+      { title: 'Create Date', field: 'date' },
+      { title: 'Reporter Account', field: 'account' },
+      { title: 'Communication Content', field: 'communication' }
+    ],
+    data: [
+      {
+        date: 'Feb 28, 2023',
+        account: 'Howard',
+        communication:
+        'Discussed the new product launch strategy with the client and received positive feedback.' +
+        'The client agreed to place an order for 500 units of the new product. They also requested a 10% discount on the total order value.'
+      }
+    ]
+  });
+  const salesRecordContent = createContent([salesRecordTable]);
+
+  const taskManagementTable = new Kuc.ReadOnlyTable({
+    label: 'Task Management',
+    columns: [
+      { title: 'Status', field: 'status' },
+      { title: 'Task Executor', field: 'executor' },
+      { title: 'Task Requirements', field: 'requirements' },
+      { title: 'Result Reporting', field: 'reporting' }
+    ],
+    data: [
+      {
+        status: 'In Progress',
+        executor: 'Mike',
+        requirements:
+        'Conduct market research on the latest trends and consumer behavior in the target market.',
+        reporting: 'Report due by March 15th.'
+      }
+    ]
+  });
+  const taskManagementContent = createContent([taskManagementTable]);
+
+  const salesOrderTable = new Kuc.ReadOnlyTable({
+    label: 'Sales Order',
+    columns: [{ title: 'Order Number', field: 'order' }],
+    data: [{ order: '1' }]
+  });
+  const salesOrderContent = createContent([salesOrderTable]);
+
+  const tabs = createTabs(
+    [salesRecordContent, taskManagementContent, salesOrderContent],
+    true
+  );
+  addTabsToDOM(tabs, SPACE_FOR_CUSTOM_COMPONENTS);
+}
+
+function createContent(components) {
+  const content = document.createElement('div');
+  content.style.padding = '16px';
+  components.map(component => content.appendChild(component));
+  return content;
+}
+
+function createTabs(contents, isBorderVisible) {
+  const items = [
+    {
+      label: 'Sales Record',
+      value: 'salesRecordTab',
+      content: contents ? contents[0] : ''
+    },
+    {
+      label: 'Task Management',
+      value: 'taskManagementTab',
+      content: contents ? contents[1] : ''
+    },
+    {
+      label: 'Sales Order',
+      value: 'salesOrderTab',
+      content: contents ? contents[2] : ''
+    }
+  ];
+  const tabs = new Kuc.Tabs({
+    value: 'salesRecordTab',
+    borderVisible: isBorderVisible,
+    items
+  });
+  return tabs;
+}
+
+function addTabsToDOM(tabs, space_id) {
+  const spaceElement = kintone.app.record.getSpaceElement(space_id);
+  spaceElement.appendChild(tabs);
+}
+```
+
+### Initialize Kintone Native Tabs
```

Comment:
```suggestion
### Initialize Tabs for Kintone native fields
```

Reply:
And for preventing DOM manipulations, we created the `borderVisible: false` option.
So please mention that for example...
You can set Kintone native fields to Tabs content without DOM manipulation setting the `borderVisible: false` option.